.github/workflows/bundle-stats.yml

```
@@ -0,0 +1,57 @@
+        required: true
+        type: string
+    outputs:
+      stats:
+        description: "Bundle stats comparison"
+        value: ${{ jobs.bundle-stats.outputs.stats }}
+
+jobs:
+  bundle-stats:
+    runs-on: ubuntu-latest
+    outputs:
+      stats: ${{ steps.compare.outputs.stats }}
```

actionlint (1.7.4)

property "compare" is not defined in object type {gist-ops: {conclusion: string; outcome: string; outputs: object}}

(expression)